app/views/reports/regions/_header.html.erb

```diff
@@ -21,16 +21,21 @@
   <p class="mb-12px fs-xxlarge fw-bold">
     <%= @region.name %>
   </p>
+  <%
+    # We want to maintain the request params for Overview and Detail nav links, but _only_ in the
+    # case where its _not_ a quarter (as those pages don't support quarters).
+    url_params = (params.dig(:period, :type) == "quarter") ? {} : request.params
```

Comment:
Any harm in _not_ excluding quarters? Do those actions gracefully ignore unsupported params, or do they loudly fail?

Reply:
They will fail, because we try to build the `Period` from params for all the actions, and a Period can built w/ a type of `month` or `quarter`. But only the `cohort` stuff deals w/ quarters (for now).